internal/controller/stas/testdata/scan-job/expected-scan-job.yaml

```diff
@@ -12,7 +12,7 @@ metadata:
     workload.statnett.no/name: echo
     workload.statnett.no/namespace: replica-set
   namespace: image-scanner
-  name: echo-6bdfc76c56-8ae43-738e6
+  name: echo-6bdfc76c56-8ae43-38618
```

Comment:
I think the scan job name is a hash of the `Workload` struct slightly modified in this PR. This should not matter, as this is done only to get less chance of scan job name collisions.